assets/component-cart.css

```
@@ -22,20 +22,28 @@ cart-items.is-empty .title-wrapper-with-link,
 }
 
 .cart__empty-text {
-  margin: 4.5rem 0 5.5rem;
+  margin: 4.5rem 0 2rem;
 }
 
 .cart__contents > * + * {
   margin-top: 2.5rem;
 }
 
+.cart__login-title {
+  margin: 5.5rem 0 1.2rem;
+}
+
+.cart__login-paragraph a {
+  font-size: inherit;
+}
+
 @media screen and (min-width: 990px) {
   .cart__warnings {
-    padding: 10rem 0 15rem;
+    padding: 7rem 0 1rem;
```

```suggestion
    padding: 3rem 0 1rem;
```
I missed this white space adjustment, sorry! This is for smaller screens 🙌